Firebase/android/app/src/main/java/com/novoda/bonfire/channel/NewChannelActivity.java

```diff
@@ -14,13 +14,12 @@
 public class NewChannelActivity extends BaseActivity {
 
     private NewChannelPresenter newChannelPresenter;
+    private Toolbar toolbar;
```

Comment:
[iOS-probably-dummy-question] It's not clear for me why you are using this toolbar here. Where do you use it?